spec/factories/events.rb

```diff
@@ -1,5 +1,7 @@
 FactoryBot.define do
   factory :event do
-    title { "MyString" }
+    title { "EventName" }
+    date { "12/12/2019" }
```

Comment:
is this the best format for date?

Reply:
Ah, no, I've updated it to `2019-12-12` format.